### PR TITLE
LPS-107659 Renames Theme Name from Latin to English: Chameleon

### PR DESCRIPTION
**Pull request by marcoscv-work**

### CODE REVIEW

**Comment by marcoscv-work** [comment]
cc @jbalsas

File: portal-kernel/bnd.bnd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Bundle-Name: ${manifest.bundle.name}
 Bundle-SymbolicName: ${manifest.bundle.symbolic.name}
-Bundle-Version: 5.5.0
```
Comment (jbalsas) [inline portal-kernel/bnd.bnd]: @brianchandotcom, not sure if this is really needed or if we could change this in a different way :(

Comment (jbalsas) [reply portal-kernel/bnd.bnd]: Ah, I think we mixed in setting the theme by default and just commiting the theme...

**Comment by jbalsas** [comment]
ci:close

**Comment by jbalsas** [comment]
ci:stop

**Pull request closed without merging**